scripts.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-use-before-define */
+let books = [];
+const booksList = document.getElementById('book-list');
+const errorMessage = document.getElementById('error-message');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+function addBook(title, author) {
+  // Check if book already exists in collection
+  if (isDuplicate(title, author)) {
+    errorMessage.textContent = 'This book is already exist!';
+    errorMessage.style.display = 'block';
+    bookTitle.style.border = '1px solid red';
+    bookAuthor.style.border = '1px solid red';
+    setTimeout(() => {
+      errorMessage.style.display = 'none';
+      bookTitle.style.border = '';
+      bookAuthor.style.border = '';
+      clearInput();
+    }, 5000); // hide the error message after 5 seconds
+    return;
+  }
```

Comment:
Very well done by adding this verification and by setting timeout for the error message 🥇